MobileCenterCrashes/MobileCenterCrashes/Internals/Util/MSErrorLogFormatter.m

```diff
@@ -645,15 +645,15 @@ + (NSString *)anonymizedPathFromPath:(NSString *)path {
 
   if (([path length] > 0) && [path hasPrefix:@"/Users/"]) {
     NSError *error = nil;
-    NSRegularExpression *regex =
-        [NSRegularExpression regularExpressionWithPattern:@"(/Users/[^/]+/)" options:0 error:&error];
+    NSString *regexPattern = @"(/Users/[^/]+/)";
+    NSRegularExpression *regex = [NSRegularExpression regularExpressionWithPattern:regexPattern options:0 error:&error];
+    if (error) {
+      MSLogError([MSCrashes logTag], @"Couldn't create regular expression with pattern\"%@\": %@", regexPattern, error.localizedDescription);
+    }
```

Comment:
@MatkovIvan After the behavior here is stabilized (@TroubleMakerBen will probably react too), please add unit test to make sure that the anonymisation process is covered.